x-pack/plugins/endpoint/public/components/endpoints_page.tsx

```diff
@@ -51,22 +51,27 @@ const columns = [
     render: (name: string, item: { _id: string }) => {
       return <EndpointName name={name} path={`/endpoints/view/${item._id}`} />;
     },
+    'data-test-subj': 'indexTableCell-name',
```

Comment:
While reviewing the style guide https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md I caught that 'data-test-subj' should be all camelcase except for the case where the hyphenated portion is dynamically generated.

Reply:
Thanks! Fixed.